superset/assets/spec/javascripts/explorev2/components/CheckboxControl_spec.jsx

```diff
@@ -4,29 +4,24 @@ import { Checkbox } from 'react-bootstrap';
 import sinon from 'sinon';
 import { expect } from 'chai';
 import { describe, it, beforeEach } from 'mocha';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 
 import CheckboxControl from '../../../../javascripts/explorev2/components/controls/CheckboxControl';
 
 const defaultProps = {
   name: 'show_legend',
   onChange: sinon.spy(),
+  value: false,
 };
 
 describe('CheckboxControl', () => {
   let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(<CheckboxControl {...defaultProps} />);
+    wrapper = mount(<CheckboxControl {...defaultProps} />);
   });
 
   it('renders a Checkbox', () => {
     expect(wrapper.find(Checkbox)).to.have.lengthOf(1);
   });
-
-  it('calls onChange when toggled', () => {
```

Comment:
Somehow couldn't get simulate to work in the context of a mount (instead of shallow). Gave up after loosing much more time than that test is worth...